src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -764,17 +611,6 @@ export default {
 				})
 
 				this.searchResults = response?.data?.ocs?.data || []
```

Comment:
Filter existing 1-1 results once instead of iterating for each search results:

```js
const oneToOneMap = this.conversationsList.reduce((acc, conv) => {
  if (conv.type === CONVERSATION.TYPE.ONE_TO_ONE) {
    acc.push(c.name)
  }
  return acc
}, [this.$store.getters.getUserId()])

this.searchResults = response?.data?.ocs?.data.filter((match) => {
	return !(match.source === ATTENDEE.ACTOR_TYPE.USERS && oneToOneMap.includes(match.id))
}) ?? []
```